ccl/changefeedccl: TestChangefeedTimelyResolvedTimestampUpdatePostRollingRestart failed

ccl/changefeedccl.TestChangefeedTimelyResolvedTimestampUpdatePostRollingRestart [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/17555849?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/17555849?buildTab=artifacts#/) on master @ [6c92b92b20bd46e71a9c3deacad6fe1223d5a2e2](https://github.com/cockroachdb/cockroach/commits/6c92b92b20bd46e71a9c3deacad6fe1223d5a2e2):


```
*   | 	GOROOT/src/runtime/iface.go:434
*   | github.com/cockroachdb/cockroach/pkg/ccl/changefeedccl.TestChangefeedTimelyResolvedTimestampUpdatePostRollingRestart
*   | 	pkg/ccl/changefeedccl/changefeed_test.go:6565
*   | testing.tRunner
*   | 	GOROOT/src/testing/testing.go:1689
*   | runtime.goexit
*   | 	src/runtime/asm_arm64.s:1222
* Wraps: (2) interface conversion: *changefeedccl.sinklessFeed is not cdctest.EnterpriseTestFeed: missing method Details
* Error types: (1) *withstack.withStack (2) *runtime.TypeAssertionError
*
    stopper.go:226: -- test log scope end --

ERROR: a panic has occurred!
Details cannot be printed yet because we are still unwinding.
Hopefully the test harness prints the panic below, otherwise check the test logs.

test logs left over in: /artifacts/tmp/_tmp/f236cfce7e32f4b492de4b4f1cd80c96/logTestChangefeedTimelyResolvedTimestampUpdatePostRollingRestart3899893785
    stopper.go:226: panic: interface conversion: *changefeedccl.sinklessFeed is not cdctest.EnterpriseTestFeed: missing method Details
--- FAIL: TestChangefeedTimelyResolvedTimestampUpdatePostRollingRestart (131.25s)
panic: interface conversion: *changefeedccl.sinklessFeed is not cdctest.EnterpriseTestFeed: missing method Details [recovered]
	panic: interface conversion: *changefeedccl.sinklessFeed is not cdctest.EnterpriseTestFeed: missing method Details [recovered]
	panic: interface conversion: *changefeedccl.sinklessFeed is not cdctest.EnterpriseTestFeed: missing method Details [recovered]
	panic: interface conversion: *changefeedccl.sinklessFeed is not cdctest.EnterpriseTestFeed: missing method Details [recovered]
	panic: interface conversion: *changefeedccl.sinklessFeed is not cdctest.EnterpriseTestFeed: missing method Details

goroutine 122320 [running]:
testing.tRunner.func1.2({0x5de88a0, 0x401af19680})
	GOROOT/src/testing/testing.go:1631 +0x1c4
testing.tRunner.func1()
	GOROOT/src/testing/testing.go:1634 +0x33c
panic({0x5de88a0?, 0x401af19680?})
	GOROOT/src/runtime/panic.go:770 +0x124
github.com/cockroachdb/cockroach/pkg/util/leaktest.AfterTest.func2()
	pkg/util/leaktest/leaktest.go:132 +0x370
panic({0x5de88a0?, 0x401af19680?})
	GOROOT/src/runtime/panic.go:770 +0x124
github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).recover(0x401e85c6c8?, {0x81d1748, 0xc786cc0})
	pkg/util/stop/stopper.go:226 +0x68
panic({0x5de88a0?, 0x401af19680?})
	GOROOT/src/runtime/panic.go:770 +0x124
github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).Stop(0x400bcc4ea0, {0x81d1748, 0xc786cc0})
	pkg/util/stop/stopper.go:535 +0x154
panic({0x5de88a0?, 0x401af19680?})
	GOROOT/src/runtime/panic.go:770 +0x124
github.com/cockroachdb/cockroach/pkg/ccl/changefeedccl.TestChangefeedTimelyResolvedTimestampUpdatePostRollingRestart(0x400601aea0)
	pkg/ccl/changefeedccl/changefeed_test.go:6565 +0xaf0
testing.tRunner(0x400601aea0, 0x6d49820)
	GOROOT/src/testing/testing.go:1689 +0xec
created by testing.(*T).Run in goroutine 1
	GOROOT/src/testing/testing.go:1742 +0x318
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
/cc @cockroachdb/cdc
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestChangefeedTimelyResolvedTimestampUpdatePostRollingRestart.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>


Jira issue: CRDB-43918

ccl/changefeedccl.TestChangefeedTimelyResolvedTimestampUpdatePostRollingRestart [failed](https://tanzanite.cluster.engflow.com/invocations/default/9172bad7-507a-4da7-8cd9-f513880f6ea9?testReportRun=22&testReportShard=3&testReportAttempt=1#targets-Ly9wa2cvY2NsL2NoYW5nZWZlZWRjY2w6Y2hhbmdlZmVlZGNjbF90ZXN0) on master @ [17535c13cfed95db70cd8dfb1ba6a700686f57b1](https://github.com/cockroachdb/cockroach/commits/17535c13cfed95db70cd8dfb1ba6a700686f57b1):

Fatal error:

```
panic: interface conversion: *changefeedccl.sinklessFeed is not cdctest.EnterpriseTestFeed: missing method Details [recovered]
	panic: interface conversion: *changefeedccl.sinklessFeed is not cdctest.EnterpriseTestFeed: missing method Details [recovered]
	panic: interface conversion: *changefeedccl.sinklessFeed is not cdctest.EnterpriseTestFeed: missing method Details [recovered]
	panic: interface conversion: *changefeedccl.sinklessFeed is not cdctest.EnterpriseTestFeed: missing method Details [recovered]
	panic: interface conversion: *changefeedccl.sinklessFeed is not cdctest.EnterpriseTestFeed: missing method Details

```
Stack: 

```
goroutine 164855 [running]:
testing.tRunner.func1.2({0x669b5a0, 0xc01254c690})
	GOROOT/src/testing/testing.go:1631 +0x24a
testing.tRunner.func1()
	GOROOT/src/testing/testing.go:1634 +0x377
panic({0x669b5a0?, 0xc01254c690?})
	GOROOT/src/runtime/panic.go:770 +0x132
github.com/cockroachdb/cockroach/pkg/util/leaktest.AfterTest.func2()
	pkg/util/leaktest/leaktest.go:132 +0x38a
panic({0x669b5a0?, 0xc01254c690?})
	GOROOT/src/runtime/panic.go:770 +0x132
github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).recover(0x49c6e5?, {0x8a8c8b8, 0xd5187c0})
	pkg/util/stop/stopper.go:226 +0x65
panic({0x669b5a0?, 0xc01254c690?})
	GOROOT/src/runtime/panic.go:770 +0x132
github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).Stop(0xc00b7babd0, {0x8a8c8b8, 0xd5187c0})
	pkg/util/stop/stopper.go:535 +0x170
panic({0x669b5a0?, 0xc01254c690?})
	GOROOT/src/runtime/panic.go:770 +0x132
github.com/cockroachdb/cockroach/pkg/ccl/changefeedccl.TestChangefeedTimelyResolvedTimestampUpdatePostRollingRestart(0xc007410ea0)
	pkg/ccl/changefeedccl/changefeed_test.go:6565 +0xd45
testing.tRunner(0xc007410ea0, 0x75fc038)
	GOROOT/src/testing/testing.go:1689 +0xfb
created by testing.(*T).Run in goroutine 1
	GOROOT/src/testing/testing.go:1742 +0x390
```
<details><summary>Log preceding fatal error</summary>
<p>

```
=== RUN   TestChangefeedTimelyResolvedTimestampUpdatePostRollingRestart
    test_log_scope.go:165: test logs captured to: outputs.zip/logTestChangefeedTimelyResolvedTimestampUpdatePostRollingRestart211449418
    test_log_scope.go:76: use -show-logs to present logs inline
    helpers_test.go:962: making sinkless feed factory
    helpers_test.go:1027: pgURL sinkless root
    helpers_test.go:1027: pgURL sinkless root
    testfeed_test.go:270: sinkless feed creating changefeed: CREATE CHANGEFEED FOR d.foo WITH cursor=$1, end_time=$2
    testfeed_test.go:277: closing sinkless feed
*
* ERROR: a panic has occurred!
* interface conversion: *changefeedccl.sinklessFeed is not cdctest.EnterpriseTestFeed: missing method Details
* (1) attached stack trace
*   -- stack trace:
*   | runtime.gopanic
*   | 	GOROOT/src/runtime/panic.go:770
*   | github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).Stop
*   | 	pkg/util/stop/stopper.go:535
*   | runtime.gopanic
*   | 	GOROOT/src/runtime/panic.go:770
*   | runtime.getitab
*   | 	GOROOT/src/runtime/iface.go:93
*   | runtime.typeAssert
*   | 	GOROOT/src/runtime/iface.go:434
*   | github.com/cockroachdb/cockroach/pkg/ccl/changefeedccl.TestChangefeedTimelyResolvedTimestampUpdatePostRollingRestart
*   | 	pkg/ccl/changefeedccl/changefeed_test.go:6565
*   | testing.tRunner
*   | 	GOROOT/src/testing/testing.go:1689
*   | runtime.goexit
*   | 	src/runtime/asm_amd64.s:1695
* Wraps: (2) interface conversion: *changefeedccl.sinklessFeed is not cdctest.EnterpriseTestFeed: missing method Details
* Error types: (1) *withstack.withStack (2) *runtime.TypeAssertionError
*
    stopper.go:226: -- test log scope end --

ERROR: a panic has occurred!
Details cannot be printed yet because we are still unwinding.
Hopefully the test harness prints the panic below, otherwise check the test logs.

test logs left over in: outputs.zip/logTestChangefeedTimelyResolvedTimestampUpdatePostRollingRestart211449418
    stopper.go:226: panic: interface conversion: *changefeedccl.sinklessFeed is not cdctest.EnterpriseTestFeed: missing method Details
--- FAIL: TestChangefeedTimelyResolvedTimestampUpdatePostRollingRestart (71.03s)
```
</p>
</details>

Parameters:
 - <code>attempt=1</code>
 - <code>run=22</code>
 - <code>shard=3</code>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestChangefeedTimelyResolvedTimestampUpdatePostRollingRestart.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

ccl/changefeedccl.TestChangefeedTimelyResolvedTimestampUpdatePostRollingRestart [failed](https://tanzanite.cluster.engflow.com/invocations/default/09b86016-ea5b-4088-b159-d74215970441?testReportRun=2&testReportShard=3&testReportAttempt=1#targets-Ly9wa2cvY2NsL2NoYW5nZWZlZWRjY2w6Y2hhbmdlZmVlZGNjbF90ZXN0) on master @ [17535c13cfed95db70cd8dfb1ba6a700686f57b1](https://github.com/cockroachdb/cockroach/commits/17535c13cfed95db70cd8dfb1ba6a700686f57b1):


```
=== RUN   TestChangefeedTimelyResolvedTimestampUpdatePostRollingRestart
    test_log_scope.go:165: test logs captured to: outputs.zip/logTestChangefeedTimelyResolvedTimestampUpdatePostRollingRestart2818018587
    test_log_scope.go:76: use -show-logs to present logs inline
    changefeed_test.go:6520: error executing 'ALTER TABLE d.foo SPLIT AT (SELECT * FROM generate_series(1, 1000));': pq: replica unavailable: (n3,s3):3 unable to serve request to r337:/{Table/106/1/251-Max} [(n1,s1):1, (n2,s2):2, (n3,s3):3, next=4, gen=255, sticky=9223372036.854775807,2147483647]: closed timestamp: 1730543341.491113390,0 (2024-11-02 10:29:01); raft status: {"id":"3","term":17,"vote":"3","commit":10,"lead":"0","leadEpoch":"0","raftState":"StateCandidate","applied":10,"progress":{},"leadtransferee":"0"}: have been waiting 63.00s for slow proposal RequestLease [/Table/106/1/251]
    panic.go:626: -- test log scope end --
test logs left over in: outputs.zip/logTestChangefeedTimelyResolvedTimestampUpdatePostRollingRestart2818018587
--- FAIL: TestChangefeedTimelyResolvedTimestampUpdatePostRollingRestart (859.73s)
```

Parameters:
 - <code>attempt=1</code>
 - <code>race=true</code>
 - <code>run=2</code>
 - <code>shard=3</code>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestChangefeedTimelyResolvedTimestampUpdatePostRollingRestart.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

ccl/changefeedccl.TestChangefeedTimelyResolvedTimestampUpdatePostRollingRestart [failed](https://tanzanite.cluster.engflow.com/invocations/default/322e9905-10f3-41cd-b922-65c97fe723cd?testReportRun=2&testReportShard=3&testReportAttempt=1#targets-Ly9wa2cvY2NsL2NoYW5nZWZlZWRjY2w6Y2hhbmdlZmVlZGNjbF90ZXN0) on master @ [6bb6dc96ebf0ee2f23c5c568fa0d421019dc0946](https://github.com/cockroachdb/cockroach/commits/6bb6dc96ebf0ee2f23c5c568fa0d421019dc0946):

Fatal error:

```
panic: interface conversion: *changefeedccl.sinklessFeed is not cdctest.EnterpriseTestFeed: missing method Details [recovered]
	panic: interface conversion: *changefeedccl.sinklessFeed is not cdctest.EnterpriseTestFeed: missing method Details [recovered]
	panic: interface conversion: *changefeedccl.sinklessFeed is not cdctest.EnterpriseTestFeed: missing method Details [recovered]
	panic: interface conversion: *changefeedccl.sinklessFeed is not cdctest.EnterpriseTestFeed: missing method Details [recovered]
	panic: interface conversion: *changefeedccl.sinklessFeed is not cdctest.EnterpriseTestFeed: missing method Details

```
Stack: 

```
goroutine 14115234 [running]:
testing.tRunner.func1.2({0x66a9160, 0xc02e2e8c60})
	GOROOT/src/testing/testing.go:1631 +0x24a
testing.tRunner.func1()
	GOROOT/src/testing/testing.go:1634 +0x377
panic({0x66a9160?, 0xc02e2e8c60?})
	GOROOT/src/runtime/panic.go:770 +0x132
github.com/cockroachdb/cockroach/pkg/util/leaktest.AfterTest.func2()
	pkg/util/leaktest/leaktest.go:132 +0x38a
panic({0x66a9160?, 0xc02e2e8c60?})
	GOROOT/src/runtime/panic.go:770 +0x132
github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).recover(0x49c6e5?, {0x8a91bf8, 0xd5108c0})
	pkg/util/stop/stopper.go:226 +0x65
panic({0x66a9160?, 0xc02e2e8c60?})
	GOROOT/src/runtime/panic.go:770 +0x132
github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).Stop(0xc00c175320, {0x8a91bf8, 0xd5108c0})
	pkg/util/stop/stopper.go:535 +0x170
panic({0x66a9160?, 0xc02e2e8c60?})
	GOROOT/src/runtime/panic.go:770 +0x132
github.com/cockroachdb/cockroach/pkg/ccl/changefeedccl.TestChangefeedTimelyResolvedTimestampUpdatePostRollingRestart(0xc00fdee000)
	pkg/ccl/changefeedccl/changefeed_test.go:6565 +0xd45
testing.tRunner(0xc00fdee000, 0x76084c0)
	GOROOT/src/testing/testing.go:1689 +0xfb
created by testing.(*T).Run in goroutine 1
	GOROOT/src/testing/testing.go:1742 +0x390
```
<details><summary>Log preceding fatal error</summary>
<p>

```
=== RUN   TestChangefeedTimelyResolvedTimestampUpdatePostRollingRestart
    test_log_scope.go:165: test logs captured to: outputs.zip/logTestChangefeedTimelyResolvedTimestampUpdatePostRollingRestart1263247640
    test_log_scope.go:76: use -show-logs to present logs inline
    helpers_test.go:962: making sinkless feed factory
    helpers_test.go:1027: pgURL sinkless root
    helpers_test.go:1027: pgURL sinkless root
    testfeed_test.go:270: sinkless feed creating changefeed: CREATE CHANGEFEED FOR d.foo WITH cursor=$1, end_time=$2
    testfeed_test.go:277: closing sinkless feed
*
* ERROR: a panic has occurred!
* interface conversion: *changefeedccl.sinklessFeed is not cdctest.EnterpriseTestFeed: missing method Details
* (1) attached stack trace
*   -- stack trace:
*   | runtime.gopanic
*   | 	GOROOT/src/runtime/panic.go:770
*   | github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).Stop
*   | 	pkg/util/stop/stopper.go:535
*   | runtime.gopanic
*   | 	GOROOT/src/runtime/panic.go:770
*   | runtime.getitab
*   | 	GOROOT/src/runtime/iface.go:93
*   | runtime.typeAssert
*   | 	GOROOT/src/runtime/iface.go:434
*   | github.com/cockroachdb/cockroach/pkg/ccl/changefeedccl.TestChangefeedTimelyResolvedTimestampUpdatePostRollingRestart
*   | 	pkg/ccl/changefeedccl/changefeed_test.go:6565
*   | testing.tRunner
*   | 	GOROOT/src/testing/testing.go:1689
*   | runtime.goexit
*   | 	src/runtime/asm_amd64.s:1695
* Wraps: (2) interface conversion: *changefeedccl.sinklessFeed is not cdctest.EnterpriseTestFeed: missing method Details
* Error types: (1) *withstack.withStack (2) *runtime.TypeAssertionError
*
    stopper.go:226: -- test log scope end --

ERROR: a panic has occurred!
Details cannot be printed yet because we are still unwinding.
Hopefully the test harness prints the panic below, otherwise check the test logs.

test logs left over in: outputs.zip/logTestChangefeedTimelyResolvedTimestampUpdatePostRollingRestart1263247640
    stopper.go:226: panic: interface conversion: *changefeedccl.sinklessFeed is not cdctest.EnterpriseTestFeed: missing method Details
--- FAIL: TestChangefeedTimelyResolvedTimestampUpdatePostRollingRestart (330.81s)
```
</p>
</details>

Parameters:
 - <code>attempt=1</code>
 - <code>deadlock=true</code>
 - <code>run=2</code>
 - <code>shard=3</code>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #134148 ccl/changefeedccl: TestChangefeedTimelyResolvedTimestampUpdatePostRollingRestart failed [A-cdc C-test-failure O-robot T-cdc branch-release-24.3 release-blocker]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestChangefeedTimelyResolvedTimestampUpdatePostRollingRestart.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

ccl/changefeedccl.TestChangefeedTimelyResolvedTimestampUpdatePostRollingRestart [failed](https://tanzanite.cluster.engflow.com/invocations/default/0e83f18c-77f2-413e-96f7-d8e1e5218580?testReportRun=2&testReportShard=3&testReportAttempt=1#targets-Ly9wa2cvY2NsL2NoYW5nZWZlZWRjY2w6Y2hhbmdlZmVlZGNjbF90ZXN0) on master @ [6bb6dc96ebf0ee2f23c5c568fa0d421019dc0946](https://github.com/cockroachdb/cockroach/commits/6bb6dc96ebf0ee2f23c5c568fa0d421019dc0946):

Fatal error:

```
panic: interface conversion: *changefeedccl.sinklessFeed is not cdctest.EnterpriseTestFeed: missing method Details [recovered]
	panic: interface conversion: *changefeedccl.sinklessFeed is not cdctest.EnterpriseTestFeed: missing method Details [recovered]
	panic: interface conversion: *changefeedccl.sinklessFeed is not cdctest.EnterpriseTestFeed: missing method Details [recovered]
	panic: interface conversion: *changefeedccl.sinklessFeed is not cdctest.EnterpriseTestFeed: missing method Details [recovered]
	panic: interface conversion: *changefeedccl.sinklessFeed is not cdctest.EnterpriseTestFeed: missing method Details

```
Stack: 

```
goroutine 95359 [running]:
testing.tRunner.func1.2({0x669c9a0, 0xc001981f20})
	GOROOT/src/testing/testing.go:1631 +0x24a
testing.tRunner.func1()
	GOROOT/src/testing/testing.go:1634 +0x377
panic({0x669c9a0?, 0xc001981f20?})
	GOROOT/src/runtime/panic.go:770 +0x132
github.com/cockroachdb/cockroach/pkg/util/leaktest.AfterTest.func2()
	pkg/util/leaktest/leaktest.go:132 +0x38a
panic({0x669c9a0?, 0xc001981f20?})
	GOROOT/src/runtime/panic.go:770 +0x132
github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).recover(0x49c6e5?, {0x8a8e398, 0xd51b7c0})
	pkg/util/stop/stopper.go:226 +0x65
panic({0x669c9a0?, 0xc001981f20?})
	GOROOT/src/runtime/panic.go:770 +0x132
github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).Stop(0xc00cb02c60, {0x8a8e398, 0xd51b7c0})
	pkg/util/stop/stopper.go:535 +0x170
panic({0x669c9a0?, 0xc001981f20?})
	GOROOT/src/runtime/panic.go:770 +0x132
github.com/cockroachdb/cockroach/pkg/ccl/changefeedccl.TestChangefeedTimelyResolvedTimestampUpdatePostRollingRestart(0xc00d1ded00)
	pkg/ccl/changefeedccl/changefeed_test.go:6565 +0xd45
testing.tRunner(0xc00d1ded00, 0x75fd938)
	GOROOT/src/testing/testing.go:1689 +0xfb
created by testing.(*T).Run in goroutine 1
	GOROOT/src/testing/testing.go:1742 +0x390
```
<details><summary>Log preceding fatal error</summary>
<p>

```
=== RUN   TestChangefeedTimelyResolvedTimestampUpdatePostRollingRestart
    test_log_scope.go:165: test logs captured to: outputs.zip/logTestChangefeedTimelyResolvedTimestampUpdatePostRollingRestart4153371405
    test_log_scope.go:76: use -show-logs to present logs inline
    helpers_test.go:962: making sinkless feed factory
    helpers_test.go:1027: pgURL sinkless root
    helpers_test.go:1027: pgURL sinkless root
    testfeed_test.go:270: sinkless feed creating changefeed: CREATE CHANGEFEED FOR d.foo WITH cursor=$1, end_time=$2
    testfeed_test.go:277: closing sinkless feed
*
* ERROR: a panic has occurred!
* interface conversion: *changefeedccl.sinklessFeed is not cdctest.EnterpriseTestFeed: missing method Details
* (1) attached stack trace
*   -- stack trace:
*   | runtime.gopanic
*   | 	GOROOT/src/runtime/panic.go:770
*   | github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).Stop
*   | 	pkg/util/stop/stopper.go:535
*   | runtime.gopanic
*   | 	GOROOT/src/runtime/panic.go:770
*   | runtime.getitab
*   | 	GOROOT/src/runtime/iface.go:93
*   | runtime.typeAssert
*   | 	GOROOT/src/runtime/iface.go:434
*   | github.com/cockroachdb/cockroach/pkg/ccl/changefeedccl.TestChangefeedTimelyResolvedTimestampUpdatePostRollingRestart
*   | 	pkg/ccl/changefeedccl/changefeed_test.go:6565
*   | testing.tRunner
*   | 	GOROOT/src/testing/testing.go:1689
*   | runtime.goexit
*   | 	src/runtime/asm_amd64.s:1695
* Wraps: (2) interface conversion: *changefeedccl.sinklessFeed is not cdctest.EnterpriseTestFeed: missing method Details
* Error types: (1) *withstack.withStack (2) *runtime.TypeAssertionError
*
    stopper.go:226: -- test log scope end --

ERROR: a panic has occurred!
Details cannot be printed yet because we are still unwinding.
Hopefully the test harness prints the panic below, otherwise check the test logs.

test logs left over in: outputs.zip/logTestChangefeedTimelyResolvedTimestampUpdatePostRollingRestart4153371405
    stopper.go:226: panic: interface conversion: *changefeedccl.sinklessFeed is not cdctest.EnterpriseTestFeed: missing method Details
--- FAIL: TestChangefeedTimelyResolvedTimestampUpdatePostRollingRestart (139.94s)
```
</p>
</details>

Parameters:
 - <code>attempt=1</code>
 - <code>run=2</code>
 - <code>shard=3</code>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #134148 ccl/changefeedccl: TestChangefeedTimelyResolvedTimestampUpdatePostRollingRestart failed [A-cdc C-test-failure O-robot T-cdc branch-release-24.3 release-blocker]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestChangefeedTimelyResolvedTimestampUpdatePostRollingRestart.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

ccl/changefeedccl.TestChangefeedTimelyResolvedTimestampUpdatePostRollingRestart [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/17573487?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/17573487?buildTab=artifacts#/) on master @ [0dd3c936001f8ce21c5935a641fd772d30b5c92c](https://github.com/cockroachdb/cockroach/commits/0dd3c936001f8ce21c5935a641fd772d30b5c92c):


```
*   | 	GOROOT/src/runtime/iface.go:434
*   | github.com/cockroachdb/cockroach/pkg/ccl/changefeedccl.TestChangefeedTimelyResolvedTimestampUpdatePostRollingRestart
*   | 	pkg/ccl/changefeedccl/changefeed_test.go:6565
*   | testing.tRunner
*   | 	GOROOT/src/testing/testing.go:1689
*   | runtime.goexit
*   | 	src/runtime/asm_arm64.s:1222
* Wraps: (2) interface conversion: *changefeedccl.sinklessFeed is not cdctest.EnterpriseTestFeed: missing method Details
* Error types: (1) *withstack.withStack (2) *runtime.TypeAssertionError
*
    stopper.go:226: -- test log scope end --

ERROR: a panic has occurred!
Details cannot be printed yet because we are still unwinding.
Hopefully the test harness prints the panic below, otherwise check the test logs.

test logs left over in: /artifacts/tmp/_tmp/f236cfce7e32f4b492de4b4f1cd80c96/logTestChangefeedTimelyResolvedTimestampUpdatePostRollingRestart2243528410
    stopper.go:226: panic: interface conversion: *changefeedccl.sinklessFeed is not cdctest.EnterpriseTestFeed: missing method Details
--- FAIL: TestChangefeedTimelyResolvedTimestampUpdatePostRollingRestart (107.05s)
panic: interface conversion: *changefeedccl.sinklessFeed is not cdctest.EnterpriseTestFeed: missing method Details [recovered]
	panic: interface conversion: *changefeedccl.sinklessFeed is not cdctest.EnterpriseTestFeed: missing method Details [recovered]
	panic: interface conversion: *changefeedccl.sinklessFeed is not cdctest.EnterpriseTestFeed: missing method Details [recovered]
	panic: interface conversion: *changefeedccl.sinklessFeed is not cdctest.EnterpriseTestFeed: missing method Details [recovered]
	panic: interface conversion: *changefeedccl.sinklessFeed is not cdctest.EnterpriseTestFeed: missing method Details

goroutine 94833 [running]:
testing.tRunner.func1.2({0x5de9e40, 0x4023840270})
	GOROOT/src/testing/testing.go:1631 +0x1c4
testing.tRunner.func1()
	GOROOT/src/testing/testing.go:1634 +0x33c
panic({0x5de9e40?, 0x4023840270?})
	GOROOT/src/runtime/panic.go:770 +0x124
github.com/cockroachdb/cockroach/pkg/util/leaktest.AfterTest.func2()
	pkg/util/leaktest/leaktest.go:132 +0x370
panic({0x5de9e40?, 0x4023840270?})
	GOROOT/src/runtime/panic.go:770 +0x124
github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).recover(0x401b6386c8?, {0x81d3548, 0xc786d40})
	pkg/util/stop/stopper.go:226 +0x68
panic({0x5de9e40?, 0x4023840270?})
	GOROOT/src/runtime/panic.go:770 +0x124
github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).Stop(0x400c3cb050, {0x81d3548, 0xc786d40})
	pkg/util/stop/stopper.go:535 +0x154
panic({0x5de9e40?, 0x4023840270?})
	GOROOT/src/runtime/panic.go:770 +0x124
github.com/cockroachdb/cockroach/pkg/ccl/changefeedccl.TestChangefeedTimelyResolvedTimestampUpdatePostRollingRestart(0x401746a000)
	pkg/ccl/changefeedccl/changefeed_test.go:6565 +0xaf0
testing.tRunner(0x401746a000, 0x6d4b360)
	GOROOT/src/testing/testing.go:1689 +0xec
created by testing.(*T).Run in goroutine 1
	GOROOT/src/testing/testing.go:1742 +0x318
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #134148 ccl/changefeedccl: TestChangefeedTimelyResolvedTimestampUpdatePostRollingRestart failed [A-cdc C-test-failure O-robot T-cdc branch-release-24.3 release-blocker]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestChangefeedTimelyResolvedTimestampUpdatePostRollingRestart.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

ccl/changefeedccl.TestChangefeedTimelyResolvedTimestampUpdatePostRollingRestart [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/17577967?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/17577967?buildTab=artifacts#/) on master @ [27c521de897105cdeeed88c3a853380c14345a22](https://github.com/cockroachdb/cockroach/commits/27c521de897105cdeeed88c3a853380c14345a22):


```
*   | 	GOROOT/src/runtime/iface.go:434
*   | github.com/cockroachdb/cockroach/pkg/ccl/changefeedccl.TestChangefeedTimelyResolvedTimestampUpdatePostRollingRestart
*   | 	pkg/ccl/changefeedccl/changefeed_test.go:6565
*   | testing.tRunner
*   | 	GOROOT/src/testing/testing.go:1689
*   | runtime.goexit
*   | 	src/runtime/asm_arm64.s:1222
* Wraps: (2) interface conversion: *changefeedccl.sinklessFeed is not cdctest.EnterpriseTestFeed: missing method Details
* Error types: (1) *withstack.withStack (2) *runtime.TypeAssertionError
*
    stopper.go:226: -- test log scope end --

ERROR: a panic has occurred!
Details cannot be printed yet because we are still unwinding.
Hopefully the test harness prints the panic below, otherwise check the test logs.

test logs left over in: /artifacts/tmp/_tmp/f236cfce7e32f4b492de4b4f1cd80c96/logTestChangefeedTimelyResolvedTimestampUpdatePostRollingRestart3347159134
    stopper.go:226: panic: interface conversion: *changefeedccl.sinklessFeed is not cdctest.EnterpriseTestFeed: missing method Details
--- FAIL: TestChangefeedTimelyResolvedTimestampUpdatePostRollingRestart (155.39s)
panic: interface conversion: *changefeedccl.sinklessFeed is not cdctest.EnterpriseTestFeed: missing method Details [recovered]
	panic: interface conversion: *changefeedccl.sinklessFeed is not cdctest.EnterpriseTestFeed: missing method Details [recovered]
	panic: interface conversion: *changefeedccl.sinklessFeed is not cdctest.EnterpriseTestFeed: missing method Details [recovered]
	panic: interface conversion: *changefeedccl.sinklessFeed is not cdctest.EnterpriseTestFeed: missing method Details [recovered]
	panic: interface conversion: *changefeedccl.sinklessFeed is not cdctest.EnterpriseTestFeed: missing method Details

goroutine 127877 [running]:
testing.tRunner.func1.2({0x5de9dc0, 0x40226915f0})
	GOROOT/src/testing/testing.go:1631 +0x1c4
testing.tRunner.func1()
	GOROOT/src/testing/testing.go:1634 +0x33c
panic({0x5de9dc0?, 0x40226915f0?})
	GOROOT/src/runtime/panic.go:770 +0x124
github.com/cockroachdb/cockroach/pkg/util/leaktest.AfterTest.func2()
	pkg/util/leaktest/leaktest.go:132 +0x370
panic({0x5de9dc0?, 0x40226915f0?})
	GOROOT/src/runtime/panic.go:770 +0x124
github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).recover(0x40182be6c8?, {0x81d30a8, 0xc786d40})
	pkg/util/stop/stopper.go:226 +0x68
panic({0x5de9dc0?, 0x40226915f0?})
	GOROOT/src/runtime/panic.go:770 +0x124
github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).Stop(0x400374d0e0, {0x81d30a8, 0xc786d40})
	pkg/util/stop/stopper.go:535 +0x154
panic({0x5de9dc0?, 0x40226915f0?})
	GOROOT/src/runtime/panic.go:770 +0x124
github.com/cockroachdb/cockroach/pkg/ccl/changefeedccl.TestChangefeedTimelyResolvedTimestampUpdatePostRollingRestart(0x40188e0ea0)
	pkg/ccl/changefeedccl/changefeed_test.go:6565 +0xaf0
testing.tRunner(0x40188e0ea0, 0x6d4b3b8)
	GOROOT/src/testing/testing.go:1689 +0xec
created by testing.(*T).Run in goroutine 1
	GOROOT/src/testing/testing.go:1742 +0x318
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #134148 ccl/changefeedccl: TestChangefeedTimelyResolvedTimestampUpdatePostRollingRestart failed [A-cdc C-test-failure O-robot T-cdc branch-release-24.3 release-blocker]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestChangefeedTimelyResolvedTimestampUpdatePostRollingRestart.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

ccl/changefeedccl.TestChangefeedTimelyResolvedTimestampUpdatePostRollingRestart [failed](https://tanzanite.cluster.engflow.com/invocations/default/d4b2bd6d-5bc7-4c99-99ce-db00742f83ce?testReportRun=8&testReportShard=3&testReportAttempt=1#targets-Ly9wa2cvY2NsL2NoYW5nZWZlZWRjY2w6Y2hhbmdlZmVlZGNjbF90ZXN0) on master @ [27c521de897105cdeeed88c3a853380c14345a22](https://github.com/cockroachdb/cockroach/commits/27c521de897105cdeeed88c3a853380c14345a22):

Fatal error:

```
panic: interface conversion: *changefeedccl.sinklessFeed is not cdctest.EnterpriseTestFeed: missing method Details [recovered]
	panic: interface conversion: *changefeedccl.sinklessFeed is not cdctest.EnterpriseTestFeed: missing method Details [recovered]
	panic: interface conversion: *changefeedccl.sinklessFeed is not cdctest.EnterpriseTestFeed: missing method Details [recovered]
	panic: interface conversion: *changefeedccl.sinklessFeed is not cdctest.EnterpriseTestFeed: missing method Details [recovered]
	panic: interface conversion: *changefeedccl.sinklessFeed is not cdctest.EnterpriseTestFeed: missing method Details

```
Stack: 

```
goroutine 154451 [running]:
testing.tRunner.func1.2({0x669be00, 0xc0209c9bf0})
	GOROOT/src/testing/testing.go:1631 +0x24a
testing.tRunner.func1()
	GOROOT/src/testing/testing.go:1634 +0x377
panic({0x669be00?, 0xc0209c9bf0?})
	GOROOT/src/runtime/panic.go:770 +0x132
github.com/cockroachdb/cockroach/pkg/util/leaktest.AfterTest.func2()
	pkg/util/leaktest/leaktest.go:132 +0x38a
panic({0x669be00?, 0xc0209c9bf0?})
	GOROOT/src/runtime/panic.go:770 +0x132
github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).recover(0x49c6e5?, {0x8a8d3b8, 0xd51a7c0})
	pkg/util/stop/stopper.go:226 +0x65
panic({0x669be00?, 0xc0209c9bf0?})
	GOROOT/src/runtime/panic.go:770 +0x132
github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).Stop(0xc0231c4990, {0x8a8d3b8, 0xd51a7c0})
	pkg/util/stop/stopper.go:535 +0x170
panic({0x669be00?, 0xc0209c9bf0?})
	GOROOT/src/runtime/panic.go:770 +0x132
github.com/cockroachdb/cockroach/pkg/ccl/changefeedccl.TestChangefeedTimelyResolvedTimestampUpdatePostRollingRestart(0xc0221d84e0)
	pkg/ccl/changefeedccl/changefeed_test.go:6565 +0xd45
testing.tRunner(0xc0221d84e0, 0x75fce78)
	GOROOT/src/testing/testing.go:1689 +0xfb
created by testing.(*T).Run in goroutine 1
	GOROOT/src/testing/testing.go:1742 +0x390
```
<details><summary>Log preceding fatal error</summary>
<p>

```
=== RUN   TestChangefeedTimelyResolvedTimestampUpdatePostRollingRestart
    test_log_scope.go:165: test logs captured to: outputs.zip/logTestChangefeedTimelyResolvedTimestampUpdatePostRollingRestart4254649745
    test_log_scope.go:76: use -show-logs to present logs inline
    helpers_test.go:962: making sinkless feed factory
    helpers_test.go:1027: pgURL sinkless root
    helpers_test.go:1027: pgURL sinkless root
    testfeed_test.go:270: sinkless feed creating changefeed: CREATE CHANGEFEED FOR d.foo WITH cursor=$1, end_time=$2
    testfeed_test.go:277: closing sinkless feed
*
* ERROR: a panic has occurred!
* interface conversion: *changefeedccl.sinklessFeed is not cdctest.EnterpriseTestFeed: missing method Details
* (1) attached stack trace
*   -- stack trace:
*   | runtime.gopanic
*   | 	GOROOT/src/runtime/panic.go:770
*   | github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).Stop
*   | 	pkg/util/stop/stopper.go:535
*   | runtime.gopanic
*   | 	GOROOT/src/runtime/panic.go:770
*   | runtime.getitab
*   | 	GOROOT/src/runtime/iface.go:93
*   | runtime.typeAssert
*   | 	GOROOT/src/runtime/iface.go:434
*   | github.com/cockroachdb/cockroach/pkg/ccl/changefeedccl.TestChangefeedTimelyResolvedTimestampUpdatePostRollingRestart
*   | 	pkg/ccl/changefeedccl/changefeed_test.go:6565
*   | testing.tRunner
*   | 	GOROOT/src/testing/testing.go:1689
*   | runtime.goexit
*   | 	src/runtime/asm_amd64.s:1695
* Wraps: (2) interface conversion: *changefeedccl.sinklessFeed is not cdctest.EnterpriseTestFeed: missing method Details
* Error types: (1) *withstack.withStack (2) *runtime.TypeAssertionError
*
    stopper.go:226: -- test log scope end --

ERROR: a panic has occurred!
Details cannot be printed yet because we are still unwinding.
Hopefully the test harness prints the panic below, otherwise check the test logs.

test logs left over in: outputs.zip/logTestChangefeedTimelyResolvedTimestampUpdatePostRollingRestart4254649745
    stopper.go:226: panic: interface conversion: *changefeedccl.sinklessFeed is not cdctest.EnterpriseTestFeed: missing method Details
--- FAIL: TestChangefeedTimelyResolvedTimestampUpdatePostRollingRestart (83.17s)
```
</p>
</details>

Parameters:
 - <code>attempt=1</code>
 - <code>run=8</code>
 - <code>shard=3</code>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #134148 ccl/changefeedccl: TestChangefeedTimelyResolvedTimestampUpdatePostRollingRestart failed [A-cdc C-test-failure O-robot T-cdc branch-release-24.3 release-blocker]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestChangefeedTimelyResolvedTimestampUpdatePostRollingRestart.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

ccl/changefeedccl.TestChangefeedTimelyResolvedTimestampUpdatePostRollingRestart [failed](https://tanzanite.cluster.engflow.com/invocations/default/b1ce1a73-3cf0-4f83-96db-b4e555ad6751?testReportRun=1&testReportShard=3&testReportAttempt=1#targets-Ly9wa2cvY2NsL2NoYW5nZWZlZWRjY2w6Y2hhbmdlZmVlZGNjbF90ZXN0) on master @ [546c489ae32a83b8d312977156cc8f2be93b6a49](https://github.com/cockroachdb/cockroach/commits/546c489ae32a83b8d312977156cc8f2be93b6a49):


```
=== RUN   TestChangefeedTimelyResolvedTimestampUpdatePostRollingRestart
    test_log_scope.go:165: test logs captured to: outputs.zip/logTestChangefeedTimelyResolvedTimestampUpdatePostRollingRestart3166931145
    test_log_scope.go:76: use -show-logs to present logs inline
    changefeed_test.go:6520: error executing 'ALTER TABLE d.foo SPLIT AT (SELECT * FROM generate_series(1, 1000));': pq: split at key /Table/106/1/254 failed: replica unavailable: (n2,s2):3 unable to serve request to r301:/{Table/106/1/253-Max} [(n1,s1):1, (n3,s3):2, (n2,s2):3, next=4, gen=257, sticky=9223372036.854775807,2147483647]: closed timestamp: 1730722807.971312169,0 (2024-11-04 12:20:07); raft status: {"id":"3","term":19,"vote":"2","commit":56,"lead":"0","leadEpoch":"0","raftState":"StateFollower","applied":56,"progress":{},"leadtransferee":"0"}: have been waiting 60.50s for slow proposal RequestLease [/Table/106/1/253]
    panic.go:626: -- test log scope end --
test logs left over in: outputs.zip/logTestChangefeedTimelyResolvedTimestampUpdatePostRollingRestart3166931145
--- FAIL: TestChangefeedTimelyResolvedTimestampUpdatePostRollingRestart (1682.59s)
```

Parameters:
 - <code>attempt=1</code>
 - <code>race=true</code>
 - <code>run=1</code>
 - <code>shard=3</code>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #134148 ccl/changefeedccl: TestChangefeedTimelyResolvedTimestampUpdatePostRollingRestart failed [A-cdc C-test-failure O-robot T-cdc branch-release-24.3 release-blocker]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestChangefeedTimelyResolvedTimestampUpdatePostRollingRestart.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

--- a/pkg/ccl/changefeedccl/changefeed_test.go
+++ b/pkg/ccl/changefeedccl/changefeed_test.go
@@ -6472,6 +6472,11 @@ func TestChangefeedTimelyResolvedTimestampUpdatePostRollingRestart(t *testing.T)
 	defer leaktest.AfterTest(t)()
 	defer log.Scope(t).Close(t)
 
+	// This test requires many range splits, which can be slow under certain test
+	// conditions. Skip potentially slow tests.
+	skip.UnderDeadlock(t)
+	skip.UnderRace(t)
+
 	opts := makeOptions()
 	defer addCloudStorageOptions(t, &opts)()
 	opts.forceRootUserConnection = true
@@ -6549,6 +6554,8 @@ func TestChangefeedTimelyResolvedTimestampUpdatePostRollingRestart(t *testing.T)
 	// Perform the rolling restart.
 	require.NoError(t, tc.Restart())
 
+	// For validation, the test requires an enterprise feed.
+	feedTestEnterpriseSinks(&opts)
 	sinkType := randomSinkTypeWithOptions(opts)
 	f, closeSink := makeFeedFactoryWithOptions(t, sinkType, tc, tc.ServerConn(0), opts)
 	defer closeSink()
@@ -6561,7 +6568,7 @@ func TestChangefeedTimelyResolvedTimestampUpdatePostRollingRestart(t *testing.T)
 
 	defer DiscardMessages(testFeed)()
 
-	// Ensure the changeefeed is able to complete in a reasonable amount of time.
+	// Ensure the changefeed is able to complete in a reasonable amount of time.
 	require.NoError(t, testFeed.(cdctest.EnterpriseTestFeed).WaitForStatus(func(s jobs.Status) bool {
 		return s == jobs.StatusSucceeded
 	}))

